include/gatekeeper_net.h

```diff
@@ -73,6 +74,10 @@ struct gatekeeper_if {
 	 */
 	rte_atomic16_t	rx_queue_id;
 	rte_atomic16_t	tx_queue_id;
+
+	/* Specify the IPv4 and IPv6 addresses of this interface. */
+	struct in_addr  ip4_addr;
+	struct in6_addr ip6_addr;
```

Comment:
Yes, you can.

However, if there's a hot spot that checks that for IPv6, it may be too expensive since you'll be checking that 16 bytes are zero.